webapp/templates/service-worker.js

```diff
@@ -23,16 +25,20 @@ self.addEventListener(
           if (r) {
             return r;
           }
-          return fetch(e.request)
+
+          // IMPORTANT: Clone the request to reuse in fetch.
```

Comment:
@lukebjerring according to https://developers.google.com/web/fundamentals/primers/service-workers/ , we also need to clone the request.